README.adoc

```diff
@@ -44,7 +44,7 @@ There is many other ways one can help us: packaging, communication, etc ...
  * https://github.com/apache/james-project/#how-to-release-via-maven-release-plugin[How to release via maven release plugin]
  * https://github.com/apache/james-project/#how-to-check-the-compilation[How to check the compilation]
  * https://github.com/apache/james-project/#how-to-run-james-in-docker[How to run James in Docker]
- ** https://github.com/apache/james-project/#run-james-with-java-8--guice--cassandra-rabbitmq-swift-elasticsearch[Run James with Java 8 + Guice + Cassandra + RabbitMQ + Swift + ElasticSearch]
+ ** https://github.com/apache/james-project/#run-james-with-java-8--guice--cassandra--rabbitmq--swift--elasticsearch[Run James with Java 8 + Guice + Cassandra + RabbitMQ + Swift + ElasticSearch]
```

Comment:
can you try with ++ (see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links )